.github/workflows/ci.yaml

```diff
@@ -19,7 +19,5 @@ jobs:
               run: yarn lint
             - name: Check types compile
               run: yarn type-check
-            - name: Check tests pass
-              run: yarn test
```

Comment:
If this is because tests are failing because there aren't any, you can add the `--passWithNoTests` flag to to the `yarn test` scripts in `package.json`!!